src/Templates/ModelsTemplate.cshtml

```diff
@@ -108,6 +108,14 @@ func joinConst(s interface{}, sep string) string {
 	}
 	return strings.Join(ss, sep)
 }
+
+@EmptyLine
+func validateError(err error) {
+	if err != nil {
+		panic(err)

```

Comment:
out of interest why panic verses propagating the error?

Reply:
Looking through the usage, I guess maybe it makes sense when we do that casting between the types, not 100% about the others ones.

---
In reply to: [212587493](https://github.com/Azure/autorest.go/pull/147#discussion_r212587493) [](ancestors = 212587493)

Reply:
The storage folks are still debating on how they want to handle this.